CMakeLists.txt

```diff
@@ -1828,8 +1828,15 @@ endif()
 # Generate and use MCUboot related artifacts as needed.
 if(CONFIG_BOOTLOADER_MCUBOOT)
   get_target_property(signing_script zephyr_property_target SIGNING_SCRIPT)
+
   if(NOT signing_script)
-    set_target_properties(zephyr_property_target PROPERTIES SIGNING_SCRIPT ${CMAKE_CURRENT_LIST_DIR}/cmake/mcuboot.cmake)
+    zephyr_get(signing_script VAR SIGNING_SCRIPT SYSBUILD)
```

Comment:
we need to decide who should take precedence here.

Cause a Zephyr module could be setting the `SIGNING_SCRIPT` property, which, as I can see results in:
- User cannot override by setting specific `-DSIGNING_SCRIPT=<script>`
- Sysbuild setting will be ignored

That could be acceptable, but we need to make it clear to users the order of precedence here.